### ISSUE TITLE
labelsync: nightly backfill job is occasionally failing

### ISSUE THREAD

**Issue by bcoe** [googleapis/repo-automation-bots#2163]
The nightly job to sync labels seems to periodically be failing:

```bash
{
httpRequest: {
}
insertId: "12urhbdg2cyc7kl"
jsonPayload: {
@type: "type.googleapis.com/google.cloud.scheduler.logging.AttemptFinished"
jobName: "projects/repo-automation-bots/locations/us-central1/jobs/snippet-bot-sync-label"
status: "UNKNOWN"
targetType: "HTTP"
url: "https://serverless-scheduler-proxy-c7mcaca6sa-uc.a.run.app/v0"
}
logName: "projects/repo-automation-bots/logs/cloudscheduler.googleapis.com%2Fexecutions"
receiveTimestamp: "2021-06-23T10:03:00.748854553Z"
resource: {
labels: {3}
type: "cloud_scheduler_job"
}
severity: "ERROR"
timestamp: "2021-06-23T10:03:00.748854553Z"
}
```

**Comment by bcoe** [comment]
@JustinBeckwith please feel free to reassign, this is just triage.

**Comment by tmatsuo** [comment]
It must be timeout probably in the scheduler proxy because the finish time is just around 3 minutes.

2021-06-23 03:00:00.755 UTC-7
2021-06-23 03:03:00.748 UTC-7

**Comment by tmatsuo** [comment]
But I'm not sure where it's defined.

**Comment by tmatsuo** [comment]
I think I figured it out

gcf-utils@11.1.0 has a bug for not queueing the tasks in parallel and it's fixed with 11.3.0.

**Comment by tmatsuo** [comment]
Actually it's fixed with 11.2.0:
https://github.com/googleapis/repo-automation-bots/pull/2122

I'll update gcf-utils.

**Comment by tmatsuo** [comment]
It turned out that scheduler job has 180 second timeout by default.
I'll create a PR for making it 540s in cron-utils.
